### DISCUSSION TITLE
How do I make changes to ui and test it locally?

### DISCUSSION THREAD

**Discussion by graciasrochelle** [tchiotludo/akhq#1378] [Q&A] [answered]
I am new here and trying to add an enhancement. I made the change but I am not able to run the code locally or see the changes when I run docker-compose up. Could you please assists? Thanks in advance.

**Comment by tchiotludo** [comment, marked as answer]
Please follow this [tutorial here ](https://akhq.io/docs/dev.html#setup-local-dev-environment-on-windows)

**Comment by graciasrochelle** [comment]
Thanks! Managed to run it using that file
